### PR TITLE
Added Cocoa.cr

### PR DESCRIPTION
**Pull request by Iainmon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,6 +295,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [mos_game](https://github.com/bararchy/mos_game) - Mini Offline Singleplayer game
 
 ## GUI library
+ * [Cocoa.cr](https://github.com/Iainmon/Cocoa.cr) - A framework for developing macOS GUI applications using Crystal
```
Comment (veelenga) [inline README.md]: Let's omit `using Crystal` (it is implied here)

Comment (Iainmon) [reply README.md]: @veelenga Sounds good! Should I make this change, and submit another pull request? I am fairly new to pull requests, so I am not sure what the best way to make this change would be.

Comment (veelenga) [reply README.md]: No, you can just make a new commit to this PR.

Comment (Iainmon) [reply README.md]: @veelenga Just made the change!

**Comment by veelenga** [comment]
@Iainmon this project looks promising. Are you going to evolve it?

**Comment by veelenga** [comment]
Closing because of inactivity. @Iainmon please re-submit once it is ready.

**Pull request closed without merging**